pages/installation.md

```diff
@@ -134,3 +134,5 @@ The key is to append the custom LS configuration to [lspconfig](https://github.c
 
     lspconfig.lexical.setup({})
 ```
+
+If the configuration above doesn't work for you, please make sure your neovim version is `>= 0.9.1`. If it still doesn't work after upgrading the version, please try this [minimal configuration](https://github.com/scottming/nvim-mini-for-lexical).
```

Comment:
I'd call this out at the beginning. Lexical simply doesn't work in neovim < 0.9.1, correct?

Reply:
Yes, almost, it doesn't work in neovim < 0.9.0